tests/cross_fw/examples/example_scope.json

```diff
@@ -265,6 +265,7 @@
         "backend": "openvino",
         "requirements": "examples/llm_compression/openvino/smollm2_360m_fp8/requirements.txt",
         "cpu": "Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz",
+        "xfail_reason": "FP8 output regression - CVS-161459",
```

Comment:
My opinion, that all examples have to work. Please set the previous openvino version in the requirements.txt for this example.

Reply:
In any case, this example works, since there are no problems observed from the side of the NNCF API or quantization scheme.

Reply:
NNCF has separate requirements.txt for each example. Please update openvino version for "examples/llm_compression/openvino/smollm2_360m_fp8".

Reply:
As **you** wish.